chatgpt.js

```diff
@@ -1455,7 +1455,7 @@ const chatgpt = { // eslint-disable-line no-redeclare
                     );
                     response = responseDivs[nthOfResponse - 1].textContent;
                 }
-                response = response.replace(/^ChatGPTChatGPT/, ''); // strip sender name
+                response = response.replace(/^ChatGPT/, ''); // strip sender name
```

Comment:
Actually 'ChatGPTChatGPT' was required previously, did you test and find out it is no longer required?

Reply:
I tested the change works w/ new UI but not the old so I added fallback regex to accomodate both